google/generativeai/models.py

```diff
@@ -49,12 +64,25 @@ def __init__(
         self._client = client
 
     def __iter__(self):
+        """
+        Returns an iterator over the models.
+
+        Yields:
+            model_types.Model: A model object from the iterable.
```

Comment:
There are a couple of spots in this file where you've added types in the docstrings. I'd prefer if you add the types to the function definitions instead. 

Shouldn't this just be:

`def __iter__(self) -> Iterable[model_types.Model]`